src/s_stuff.h

```diff
@@ -185,6 +193,18 @@ EXTERN void sys_closesocket(int fd);
 typedef void (*t_fdpollfn)(void *ptr, int fd);
 EXTERN void sys_addpollfn(int fd, t_fdpollfn fn, void *ptr);
 EXTERN void sys_rmpollfn(int fd);
+#ifdef THREADED_IO
+#include <sys/socket.h> // struct sockaddr, socklen_t
+#define t_rbskt struct _rbskt
+typedef void (*t_fdsendrmfn)(void *ptr);
+EXTERN void sys_addpollrb(int fd, int preserve_boundaries);
+EXTERN t_rbskt* sys_getpollrb(int fd);
+void sys_addsendfdrmfn(int sockfd, t_fdsendrmfn, void* x);
+ssize_t sys_sendto(int sockfd, const void *buf, size_t len, int flags, void* addr, size_t addrlen);
```

Comment:
Let's add `sys_send` for symmetry. Also, the types of `addr` and `addrlen` should match those of `recvfrom` - or vice versa.

Reply:
edbea50b

Reply:
d0adbf69 done here for real